### DISCUSSION TITLE
Comment display has a ton of white space

### DISCUSSION THREAD

**Discussion by pattrickrice** [community/community#38207] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
![image](https://user-images.githubusercontent.com/20763171/199815077-3372c068-cf0f-4cfb-93ad-54f7020685fc.png)
GitHub for iOS, Version 1.87.0 (82263682)
Device: iPhone15,2 (iOS 16.1)
</details>

**Comment by pattrickrice** [comment]
![image](https://user-images.githubusercontent.com/20763171/199815148-61f55276-981c-454f-b7f2-14c5bc1cf389.png)

**Comment by david-engelmann** [comment]
[I checked out the issue and didn't notice the formatting problem.](https://github.com/PKief/vscode-material-icon-theme/issues/1736). Is your app up to date?

**Reply by pattrickrice** [reply]
I notice that when I reload the comment, the white space problem is gone. Maybe a transient issue?

**Comment by stevepopovich** [comment]
Hi @pattrickrice, thanks for reporting! We have seen this before and it is in our backlog to investigate. Thanks!

**Comment by amadejpapez** [comment]
Yep I am having the same issue. Very annoying. Usually when clickng on a notification that there is a new comment.
